app/frontend/src/features/profile/edit/constants.ts

```diff
@@ -0,0 +1,19 @@
+export const DEFAULT_ABOUT_ME_HEADINGS = `# Current mission
+
```

Comment:
Can you add a `<be>` to all of these so people can just click in the space and type